### PR TITLE
Add github action for welcoming first time contributor

### PR DESCRIPTION
**Pull request by chenrui333**
Add github action for welcoming first time contributor

### CODE REVIEW

File: .github/workflows/welcome_first_time_contributor.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+on: pull_request
```
Comment (dawidd6) [inline .github/workflows/welcome_first_time_contributor.yml]:
```suggestion
on:
  pull_request:
    types:
      - opened
```

**Comment by issyl0** [comment]
If we add this, I think some of the spirit of Homebrew would be gone. I know that Homebrew/homebrew-core gets a load more first time PRs, but part of what gets people coming back is the human touch - an actual human, from an actual GitHub account, saying thanks (optionally with a proportional amount of emoji). It can be a saved reply in GitHub, but the fact that a human had to click those buttons is a good thing. It feels more sincere, it gives the contributor a warm fuzzy feeling inside.

I've contributed to projects where there have been many bot messages seconds after opening my PR and I know this is only one message, but those contributions didn't make me want to do it again.

**Comment by SMillerDev** [comment]
Yeah, I oppose this too. We currently don't have welcome messages and I don't see any need for them. I think effort might be better spend on automatically adding labels like `new formula` or `build failure`

**Comment by chenrui333** [comment]
Thanks @issyl0 @SMillerDev for the valuable feedback, really appreciate it!! I will close this PR and create new action for doing the auto-labeler work. :)

Thanks @dawidd6 for improving the syntax. :)

**Pull request closed without merging**